contracts/escrow/src/state.rs

```diff
@@ -23,7 +23,7 @@ impl State {
         }
 
         if let Some(end_time) = self.end_time {
-            if env.block.time > end_time {
+            if env.block.time.nanos() > end_time {
```

Comment:
`State.end_time` is in seconds but the block time changes to nanoseconds. We need something like

```suggestion
            if env.block.time.nanos() > end_time * 1000 {
```